resources/views/layouts/app.blade.php

```diff
@@ -32,19 +32,20 @@
 
                 <div class="collapse navbar-collapse" id="navbarSupportedContent">
                     <!-- Left Side Of Navbar -->
-                    <ul class="navbar-nav mr-auto">
-                        <li><a class="navbar-brand" href="{{ route('pages.index') }}"> Pages </a> </li>
-                        <li><a class="navbar-brand" href="{{ route('menus.index') }}"> Menus </a></li>
-                    </ul>
-
+                    @if (Auth::check())
+                        <ul class="navbar-nav mr-auto">
+                            <li><a class="navbar-brand" href="{{ route('pages.index') }}"> Pages </a> </li>
+                            <li><a class="navbar-brand" href="{{ route('menus.index') }}"> Menus </a></li>
+                        </ul>
+                    @endif
                     <!-- Right Side Of Navbar -->
                     <ul class="navbar-nav ml-auto">
                         <!-- Authentication Links -->
                         @guest
                             <li class="nav-item">
                                 <a class="nav-link" href="{{ route('login') }}">{{ __('Login') }}</a>
                             </li>
-                            @if (Route::has('register'))
+                            @if (Auth::check())
```

Comment:
`Auth:check` er inne i `@guest`, så den vil aldri bli true

Du kan forresten bruke `@auth` og `@endauth` i stedet for `@if (Auth::check())`

Reply:
Endret til @auth og @endauth fra @if (Auth::check() 👍

Reply:
Did it now